### ISSUE TITLE
[Windows] Cannot save shortcuts

### ISSUE THREAD

**Issue by Drake-Knifetail** [URUWorks/TeroSubtitler#218]
**Describe the bug**
I can't save shotcuts because the "path specified cannot be found". Creating the missing folder solves the issue !

**To Reproduce**
Settings > Shortcuts > Save As...
type "preset"
![image](https://github.com/URUWorks/TeroSubtitler/assets/154935690/1b768669-425a-4d0a-9d75-d42d83c11882)

**Expected behavior**
The file (and missing folder) should be created

**Current behavior**
An error occurs and the shortcuts are not saved

**Screenshots/Video**
If applicable, add screenshots/video to help explain your problem.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version 1.0.1.3

**Additional context**

**Comment by URUWorks** [comment]
Can you try the version of the link and tell us if it is solved?

https://github.com/URUWorks/additional-files/raw/main/terosubtitler_testbuild_win64.zip

**Comment by Drake-Knifetail** [comment]
> Can you try the version of the link and tell us if it is solved?
> 
> https://github.com/URUWorks/additional-files/raw/main/terosubtitler_testbuild_win64.zip

This build indeed fixed the issue. I've tried with both the portable version you gave (which includes the Tero preset) and on the installed version I have (by replacing the files in the AppData\Local\Programs\Tero Subtitler folder and removing the shortcuts I made and the folder).

I'll note as well that the windows installer in the releases assets (https://github.com/URUWorks/TeroSubtitler/releases/tag/1.0.1.3) does not create the shortcut folder and doesn't include the Tero.key preset that is present in the zip you provided.
